src/index.js

```diff
@@ -46,11 +46,18 @@ class Radar {
     return STATUS;
   }
 
-  static initialize(publishableKey) {
+  static initialize(publishableKey, options={}) {
     if (!publishableKey) {
       console.error('Radar "initialize" was called without a publishable key');
     }
     Storage.setItem(Storage.PUBLISHABLE_KEY, publishableKey);
+    if(options){
+      let {locationTimeToLive} = options
+
+      if(locationTimeToLive && typeof(locationTimeToLive) === 'number'){
```

Comment:
Maybe we should throw a warning if it's set but not a number, but also be more lenient and accept number strings.
```js
if (locationTimeToLive) {
  const number = Number(locationTimeToLive);
  if (Number.isNaN(number)) {
    console.warn('Radar SDK: invalid number for option "locationTimeToLive"'));
  } else {
    Storage.setItem(Storage.LOCATION_TIME_TO_LIVE, locationTimeToLive)
  }
}
```

Reply:
👍